### ISSUE TITLE
Clicking the zoom in/out buttons twice crashes Ruffle (extension only)

### ISSUE THREAD

**Issue by WumboSpasm** [ruffle-rs/ruffle#1686]
If I zoom in or out twice to any level from any level on a fresh page load, the Ruffle extension crashes. This doesn't happen with the demo. Tested on Firefox Developer Edition.

[Log](https://github.com/ruffle-rs/ruffle/files/5592109/log.txt)

**Comment by parkerlreed** [comment]
What game? URL to the page/swf helps a lot in others debugging.

**Comment by WumboSpasm** [comment]
It happens with every game, you can find an example in the log file

**Comment by parkerlreed** [comment]
I'm confused because there's no zoom in/out controls in Ruffle itself. What zoom controls are you using?

**Comment by parkerlreed** [comment]
Ok I see it's browser page zoom. I see the same thing in Chromium

```
# Error Info
Error name: Error
Error message: panicked at 'called `Result::unwrap()` on an `Err` value: "WebGL: Error in renderbuffer_storage_multisample (color): 1286"', render/webgl/src/lib.rs:739:35
Error stack:
```
Error: panicked at 'called `Result::unwrap()` on an `Err` value: "WebGL: Error in renderbuffer_storage_multisample (color): 1286"', render/webgl/src/lib.rs:739:35
    at Object.j (chrome-extension://jjbcnponackegfahkmhgifneelfjdkap/dist/core.ruffle.js:1:20183)
    at __wbg_new_df6e6ab7a65c4c4d (chrome-extension://jjbcnponackegfahkmhgifneelfjdkap/dist/ruffle.js:1:31150)
    at chrome-extension://jjbcnponackegfahkmhgifneelfjdkap/dist/f14e566955e4549f1567.module.wasm:wasm-function[1036]:0x1fa480
    at chrome-extension://jjbcnponackegfahkmhgifneelfjdkap/dist/f14e566955e4549f1567.module.wasm:wasm-function[1364]:0x23a252
    at chrome-extension://jjbcnponackegfahkmhgifneelfjdkap/dist/f14e566955e4549f1567.module.wasm:wasm-function[3652]:0x2e98e5
    at chrome-extension://jjbcnponackegfahkmhgifneelfjdkap/dist/f14e566955e4549f1567.module.wasm:wasm-function[2867]:0x2cbb49
    at chrome-extension://jjbcnponackegfahkmhgifneelfjdkap/dist/f14e566955e4549f1567.module.wasm:wasm-function[3947]:0x2ee50b
    at chrome-extension://jjbcnponackegfahkmhgifneelfjdkap/dist/f14e566955e4549f1567.module.wasm:wasm-function[3976]:0x2eea40
    at chrome-extension://jjbcnponackegfahkmhgifneelfjdkap/dist/f14e566955e4549f1567.module.wasm:wasm-function[3852]:0x2ed0e8
    at chrome-extension://jjbcnponackegfahkmhgifneelfjdkap/dist/f14e566955e4549f1567.module.wasm:wasm-function[3977]:0x2eea76
```

# Player Info
Allows script access: true
Player type: Object
SWF URL: /games/data/loaders/hi_score_proxy_600x400.swf?game_path=TrailForSnail.swf
Param allowScriptAccess: sameDomain
Param movie: /games/data/loaders/hi_score_proxy_600x400.swf?game_path=TrailForSnail.swf
Param quality: high
Param bgcolor: #ff9900
Attribute 0: [object Attr]
Attribute 1: [object Attr]
Attribute 2: [object Attr]
Attribute 3: [object Attr]
Attribute 4: [object Attr]
Attribute 5: [object Attr]

# Page Info
Page URL: http://www.nick.com/games/data/spongebob/sb_trailsnail/index.html

# Browser Info
Useragent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.66 Safari/537.36
OS: Linux x86_64

# Ruffle Info
Ruffle version: 0.1.0
Ruffle source: undefined
```

**Comment by WumboSpasm** [comment]
Sorry for the confusion, I'll edit my post to clarify that

**Comment by parkerlreed** [comment]
In Chromium at least, zooming the page at any time will cause the video renderer to crash. Audio continues.

**Comment by Toad06** [comment]
I think it's possible that this is the same issue as #1264, #1538 and #1605, though this would be a different way to reproduce the bug. They all panic with a similar message:
`panicked at 'called Result::unwrap() on an Err value: "WebGL: Error in renderbuffer_storage_multisample (color): 1286"', render/webgl/src/lib.rs:739:35`

It also happens in Firefox (Windows) using the web demo: start a game, open the browser's console and stretch it to the top of the window. Ruffle will panic (I think the issue would also happen in Chrome... but it's not possible to get the console that high in this browser 😛).

**Comment by Herschel** [comment]
Probably the frame buffer size is becoming invalid (too big/small) -- we should handle this more gracefully.

**Comment by Herschel** [comment]
Fixed by e66fc6f.
